src/core/icons/fileIcons.ts

```diff
@@ -1601,6 +1601,11 @@ export const fileIcons: FileIcons = {
         '.eslintrc.base.json',
         '.eslintignore',
         '.eslintcache',
+        'eslint.config.js',
```

Comment:
Please remove the eslint.config.* entries here, we already cover them with the `FileNamePattern.Cosmiconfig`, as you can see here:

https://github.com/material-extensions/vscode-material-icon-theme/blob/bd0b2f6e8b9f07ae50bb8aa4bdefa3172d24cf01/src/core/patterns/patterns.ts#L88-L93

`eslint-options.js` you can keep here, it's fine.

Reply:
@PKief 
Thank you for letting me know!
I corrected fileName to eslint to support eslint.config.*.

https://github.com/material-extensions/vscode-material-icon-theme/pull/3156/commits/d83bf2eedcd9b6600c6d16e65f7b46870d0eb199